Kuroba/app/src/main/java/com/github/k1rakishou/chan/core/site/sites/dvach/DvachApiV2.kt

```diff
@@ -263,6 +266,21 @@ class DvachApiV2(
 
       builder.postImages(postImages, builder.postDescriptor)
 
+      if (threadPost.icon.isNotNullNorEmpty()) {
+        val document = Jsoup.parseBodyFragment(threadPost.icon)
+        val icons = document.body().select("img")
+        for (icon in icons) {
+          val imageUrl = icon?.attr("src")
+          val title = icon?.attr("title")
+
+          val iconUrl = endpoints.icon(
+            title.toString(),
+            SiteEndpoints.makeArgument("icon", imageUrl)
+          )
+          builder.addHttpIcon(ChanPostHttpIcon(iconUrl, "$title"))
```

Comment:
And then you will be able to just pass `title` as is without converting it into a string.
` builder.addHttpIcon(ChanPostHttpIcon(iconUrl, title))`